Tasks/Serzhant/Task1/Task1.py

```diff
@@ -0,0 +1,7 @@
+initial_amount = 20000
+duration = 5
+months = 12
+for i in range(duration*months):
+    initial_amount += (initial_amount * 0.15) / months
```

Comment:
the cycle is not a very good solution, since there is a specific formula for such calculations, -1 point for the unwise choice